datajoint/connection.py

```diff
@@ -57,14 +57,16 @@ class Connection:
     :param port: port number
     :param init_fun: connection initialization function (SQL)
     """
-    def __init__(self, host, user, password, port=None, init_fun=None):
+    def __init__(self, host, user, password, init_fun=None, ssl=True, port=None):
```

Comment:
```suggestion
    def __init__(self, host, user, password, *, port=None, ssl=True,  init_fun=None):
```
Since we change the order of keyword arguments, let's add the * to force people to name their arguments after host, user, and password. All following attributes must be explicitly named.

Reply:
The only reason I reorder here is because I came across a bug between the `Connection` definition and how it was initialized in `dj.conn`. We can still preserve the order here if desired. But we should at least pass a `None` for port when initialized in `dj.conn`. After discussing with @ixcat and @ttngu207, the consensus is that the least impact to the users would be to keep the args ordered and add TLS as a last arg. Can proceed this way if you agree.

Reply:
By placing the `*`after password, we make the order of arguments insignificant since they will all have to be specified by name.

Reply:
Addressed. Per discussion with the rest of the team, the preference seems to be to preserve the order and simply add option at the end to reduce impact on current users. Bug fix also applied.